src/main/java/ch/njol/skript/expressions/ExprCommand.java

```diff
@@ -67,7 +67,7 @@
 	@Override
 	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parseResult) {
 		what = matchedPattern;
-		if (!ScriptLoader.isCurrentEvent(PlayerCommandPreprocessEvent.class, ServerCommandEvent.class) && (UNKNOWN_COMMAND_SUPPORTED && !ScriptLoader.isCurrentEvent(UnknownCommandEvent.class))) {
+		if (!getParser.isCurrentEvent(PlayerCommandPreprocessEvent.class, ServerCommandEvent.class) && (UNKNOWN_COMMAND_SUPPORTED && !ScriptLoader.isCurrentEvent(UnknownCommandEvent.class))) {
```

Comment:
shouldn't this be `getParser()`

Reply:
> shouldn't this be `getParser()`
[https://github.com/SkriptLang/Skript/pull/3171/commits/17d7e58017c78d08bd6d4bb9b72f1d13103fa680](url) didn't delta fix it?

Reply:
Seems like he only fixed the first check, but not the other